src/common/containers/repository-setup.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component, PropTypes } from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import { createWebhook } from '../actions/webhook';
+import { Message } from '../components/forms';
+import Spinner from '../components/spinner';
+
+import styles from './container.css';
+
+class RepositorySetup extends Component {
```

Comment:
At first I was wondering if we need this new view at all, as maybe `createSnap` could dispatch `createWebhook` (just as `createWebhook` dispatches `createSnap` in current flow), but then I realised that creating snap goes through redirects, so having view separate from `/builds` is good choice even if ATM all it does is triggering webhook action and redirecting again to `/builds`.